pages/common/git-verify-tag.md

```diff
@@ -0,0 +1,17 @@
+# git verify-tag
+
+> Check for GPG verification of tags.
+> If tag isn't signed with -s when made, you will get an error regardless of signing.
```

Comment:
How about a shorter version:
```suggestion
> If a tag wasn't signed, an error will occur.
```

Reply:
signed manually would be better since it seems that regardless of it being signed, the -s flag seems to flag it up for git, almost as if it is treated different

Reply:
`with -s` without the full command doesn't have enough sense, also I think should be a `gitconfig` setting to auto sign tags.

Reply:
I prefer @marchersimon's suggestion — less complexity means less likelyhood of misunderstanding the conversation.

Reply:
@navarroaxel it does sign the tags with `	gpgsign = true` but not flagged for some reason

Reply:
```suggestion
> If a tag wasn't manually signed, an error will occur.
```

Reply:
@CleanMachine1 `manually` or `automatic` is not important here, @marchersimon suggestion is good enough.

Reply:
I disagree but others agree, so OK